extensions/amp-access-laterpay/0.1/laterpay-impl.js

```diff
@@ -219,12 +219,13 @@ export class LaterpayVendor {
     const urlPromise = this.accessService_.buildUrl(
       url, /* useAuthData */ false);
     return urlPromise.then(url => {
-      dev().fine(TAG, 'Authorization URL: ', url);
+      return this.accessService_.buildLoginUrl(url);
+    }).then(url => {
+      dev().info(TAG, 'Authorization URL: ', url);
       return this.timer_.timeoutPromise(
           AUTHORIZATION_TIMEOUT,
           this.xhr_.fetchJson(url, {
             credentials: 'include',
-            requireAmpResponseSourceOrigin: true,
```

Comment:
Oh, I only removed this because AMP gives me a deprecation warning:

![2017-05-16 at 10 21](https://cloud.githubusercontent.com/assets/38177/26096435/7ade6188-3a21-11e7-9bdf-34bc4b740db1.png)

I looked for info on this `ampCors` property but couldn't find anything. So then I looked for a similar [similar call](https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/0.1/amp-access-client.js#L127) on `amp-access` and just followed the same pattern.